Automattic/woocommerce-payments
# Fixes issue with adding QIT authentication if password has spaces.

Fixes #9750 

#### Changes proposed in this Pull Request









#### Prerequisites

Please read the P2 post linked for context on QIT and create the `local.env` file as specified in the instructions.

#### Testing instructions

1. On `develop`, use a password with a space in it in the `local.env` file for the `QIT_PASSWORD` variable (it doesn't need to be the correct password to see this error).
2. Run the command `npm run test:qit-security`
3. You should see an error similar to: `  No arguments expected for "partner:add" command, got "<password>".`
4. Verify that this doesn't happen on this branch. (You can check that the steps on the p2 post linked on this page work as expected).





See: PaJDYF-6n7-p2 
-------------------


**Post merge**
